easybuild/easyconfigs/s/SAMtools/SAMtools-1.7-foss-2018a-HTSlib-1.7.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+#
+# Modified for version 1.4 by: Kurt Lust, UAntwerpen
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SAMtools'
+version = '1.7'
+
+homepage = 'http://www.htslib.org/'
+description = """Samtools provide various utilities for manipulating alignments in the SAM format, 
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['e7b09673176aa32937abd80f95f432809e722f141b5342186dfef6a53df64ca1']
+
+# SAMtools needs HTSlib, but the SAMtools source contains an HTSlib source
+# tree, and it will use that by default.  This probably works okay in trivial
+# scenarios (when HTSlib is not loaded), but it causes problems when another
+# module wants to use libraries contained in both.  Doing the standard thing
+# seems likely to work better.
+htsver = '1.7'
+versionsuffix = '-HTSlib-%s' % htsver
```

Comment:
@tutufan Unless you have a particular use case for this, I wouldn't include this variant for now, and go forward with the one that uses the included `HTSlib`?